package.json

```diff
@@ -45,6 +46,7 @@
     },
     "scripts": {
         "test": "grunt test",
-        "prepare": "grunt build"
+        "prepare": "grunt build",
+        "lint": "eslint ./src"
```

Comment:
This was mostly for me to easily run `eslint` (without knowing full details of the `grunt` pipeline), and can be changed/dropped.

I will say though although, I can say that running lint via `eslint` directly way avoids an error that shows up when using `grunt eslint`:

```
(node:1224) Warning: Accessing non-existent property 'VERSION' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
```

(running with `--trace-warnings` seemed to indicate this could be originating in a dependency, `istanbul`)

Reply:
Does this lint command run it with the `.eslintrc.json`? I'm not entirely against having this avenue, but they should behave the same. In fact, the fact that you get an error one way and not the other implies there is at least some difference! Probably safest not to have two ways to do it.

Reply:
Yes, this command also uses `.eslintrc.json`, as it uses the default naming convention, and is local to the project:

> The configuration cascade works based on the location of the file being linted. If there is an .eslintrc file in the same directory as the file being linted, then that configuration takes precedence. ESLint then searches up the directory structure, merging any .eslintrc files it finds along the way until reaching either an .eslintrc file with root: true or the root directory.

For details, see [here](https://eslint.org/docs/latest/use/configure/configuration-files#cascading-and-hierarchy) and [here](https://eslint.org/docs/latest/use/configure/configuration-files#using-configuration-files)

Re: behaving the same - I am pretty sure the error is coming from one of the dependencies pulled in when the grunt tasks are run (seemingly `istanbul`, judging by the stack trace - via `--trace-warnings`): 

![image](https://user-images.githubusercontent.com/2937540/221260970-007acb79-29fc-488b-b7a6-7c456b35388c.png)

The only other difference looks to be in how the files to be linted are specified - the grunt version leverages the array with filenames, while the command here looks for all eligible files under `./src` (should ideally be the same)

That said, removing it sounds fine to me (it is done a bit differently than the other commands), though it doesn't seem like lint is being run for the [`watch` command as specified in the `CONTRIBUTING`](https://github.com/openseadragon/openseadragon/blob/master/CONTRIBUTING.md#building-from-source) doc (must run the [`watchTask` or `build`](https://github.com/openseadragon/openseadragon/blob/master/Gruntfile.js#L298) instead)